lib/kerberos.js

```diff
@@ -4,10 +4,6 @@ const kerberos = require('bindings')('kerberos');
 const KerberosClient = kerberos.KerberosClient;
 const KerberosServer = kerberos.KerberosServer;
 
-// Result Codes
-const AUTH_GSS_CONTINUE = 0;
```

Comment:
Why are these removed?

Reply:
these were throwbacks from the way the C-style api worked, each time you would call a method it would return one of these results, and then the user would use an accessor to grab the response. Instead, I chose to return the actual response or error on failure. It's analagous to:
```
client.step(..., (err, res) => {
  if (res === AUTH_GSS_ERROR) return client.getError();
  client.getResponse()
});
```
vs
```
client.step(..., (err, response) => {
  if (err) return err;
  // we already have the response ready
});
```